src/axom/quest/MeshTester.cpp

```diff
@@ -118,13 +113,12 @@ void findTriMeshIntersections(detail::UMesh* surface_mesh,
     size_t checkcount = binsToCheck.size();
     for(size_t curbin = 0; curbin < checkcount; ++curbin)
     {
-      std::vector<int> ntlist = ugrid.getBinContents(binsToCheck[curbin]);
-      std::vector<int>::iterator ntlit = ntlist.begin(), ntlend = ntlist.end();
-      for(; ntlit != ntlend; ++ntlit)
+      axom::ArrayView<int> ntlist = ugrid.getBinContents(binsToCheck[curbin]);
```

Comment:
+1 for conversion from vector to `axom::Array` and `axom::ArrayView`.

We should consider modifying the MeshTester API to remove `std::vector` in a follow-up PR.